### ISSUE TITLE
Teach minidump-processor how to grab symbols from servers

### ISSUE THREAD

**Issue by Gankra** [luser/rust-minidump#154]
Stubbing this out since I'm not sure what the scope/format is here. Whatever [mozilla-services/minidump-stackwalk](https://github.com/mozilla-services/minidump-stackwalk) does to source its symbols from (only mozilla's?) servers.

**Comment by Gankra** [comment]
Ted linked https://github.com/jrmuizel/pdb-downloader in response to this, but I was assuming we would be downloading breakpad-cfi files, since that's what dump_syms packages up?

**Comment by luser** [comment]
Yeah, I think that was me misunderstanding what you intended. As filed I believe this already works since I implemented [`HttpSymbolSupplier`](https://github.com/luser/rust-minidump/blob/25cc6ceecaff3e17a9eb9deca6658eee9240994b/breakpad-symbols/src/lib.rs#L236).

**Comment by Gankra** [comment]
Oh cool, might just be needing to hook it up in the frontend, then!
